<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetVariableDialog</class>
 <widget class="QDialog" name="SetVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border: 1px solid #222;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="variableInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="playerBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PLAYER0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER_GOOD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAYER_NEUTRAL</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Set variable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>playerBox</tabstop>
  <tabstop>variableInput</tabstop>
  <tabstop>valueInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
